components/__design__/list.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import type { CSSProperties } from 'react';
+
+export const Lists: StoryObj = {};
+
+const listContainer = {
+  padding: '1.5rem',
+} as CSSProperties;
+
+const listDarkContainer = {
```

Comment:
You don't need to do this. Storybook has dark theme by default, you can simply switch from dark to light by pressing a button...